llvm/test/MC/RISCV/rv32zalasr-valid.s

```diff
@@ -0,0 +1,70 @@
+# RUN: llvm-mc %s -triple=riscv32 -mattr=+experimental-zalasr -riscv-no-aliases -show-encoding \
```

Comment:
It's not very obvious from the test names, but I think the convention for the `-valid.s` tests is usually that the `rv32` test contains riscv32 and riscv64 RUN lines. and the `rv64` test only contains instructions that are unique to rv64. This reduces the amount of test content that is duplicated.